### PR TITLE
Update mixin to use React panels instead of Angular panels

### PR DESCRIPTION
**Pull request by Dasomeone**
A number of changes are included in this PR:
- moved libsonnet dashboards to a separate folder for archival as they rely on outdated grafonnet
- added json dashboards with modern grafana schema 
  - updated dashboards to use react-based panels

### CODE REVIEW

File: github-mixin/dashboards/libsonnet/README.md
Comment (mshahzeb) [inline github-mixin/dashboards/libsonnet/README.md:1]: Should we just drop this empty Readme file?

File: github-mixin/dashboards/api-usage.json [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+      "7d",
+      "30d"
+    ]
+  },
+  "timezone": "browser",
+  "title": "GitHub API Usage",
+  "uid": "github-api-usage",
+  "version": 2,
+  "weekStart": ""
+}
```
Comment (mshahzeb) [inline github-mixin/dashboards/api-usage.json]: Missing newline warning

**Review by mshahzeb — APPROVED** [review]
LGTM! Just some small comments.

File: github-mixin/dashboards/libsonnet/README.md
Comment (Dasomeone) [reply github-mixin/dashboards/libsonnet/README.md:1]: Oh good catch, I hadn't meant to add that!

**Pull request merged**